addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties

```diff
@@ -250,29 +250,29 @@ thing-type.config.max.shuttercontact.action-deviceDelete.option.-1 = Keine Aktio
 
 # channel types
 channel-type.max.free_mem.label = Freie Speicherpl�tze
-channel-type.max.free_mem.description = Zeigt an, wie viele freie Speicherpl�tze im MAX! Cube LAN Gateway f�r Befehle zur Verf�gung stehen.
+channel-type.max.free_mem.description = Gibt an, wie viele freie Speicherpl�tze im MAX! Cube LAN Gateway f�r Befehle zur Verf�gung stehen.
 
 channel-type.max.duty_cycle.label = Auslastungsgrad
-channel-type.max.duty_cycle.description = Zeigt den Auslastungsgrad f�r Befehle (in %) an.
+channel-type.max.duty_cycle.description = Gibt den Auslastungsgrad f�r Befehle (in %) an.
 
 channel-type.max.valve.label = Ventil
-channel-type.max.valve.description = Zeigt die Ventil�ffnung des Heizk�rperreglers (in %) an.
+channel-type.max.valve.description = Gibt die Ventil�ffnung des Heizk�rperreglers (in %) an.
 
 channel-type.max.mode.label = Modus des Ger�tes
-channel-type.max.mode.description = Zeigt den aktuellen Modus des Ger�tes an (MANUAL/AUTOMATIC/BOOST/VACATION).
+channel-type.max.mode.description = Gibt den aktuellen Modus des Ger�tes an (MANUAL/AUTOMATIC/BOOST/VACATION).
 channel-type.max.mode.state.option.AUTOMATIC = Automatisch
 channel-type.max.mode.state.option.MANUAL = Manuell
 channel-type.max.mode.state.option.BOOST = Boost
 channel-type.max.mode.state.option.VACATION = Urlaubsmodus
 
 channel-type.max.actual_temp.label = Temperatur
-channel-type.max.actual_temp.description = Zeigt die aktuell gemessene Temperatur des Heizk�rperthermostat an.
+channel-type.max.actual_temp.description = Gibt die aktuell gemessene Temperatur (in �C) des Heizk�rperthermostats an.
 
 channel-type.max.set_temp.label = Solltemperatur
-channel-type.max.set_temp.description = Erm�glicht die Steuerung der Solltemperatur des Heizk�rperthermostat.
+channel-type.max.set_temp.description = Gibt die aktuell eingestellte Solltemperatur (in �C) des Heizk�rperthermostats.
```

Comment:
Please remove the unit. It depends on the users configuration as the channel uses QuantityType.